pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/DownloadParticipantListRoute.java

```diff
@@ -94,28 +86,4 @@ protected void setResponseHeaders(Response response, String filename) {
         response.header("Content-Disposition", "attachment;filename=" + filename);
     }
 
-    /**
-     * Fetches participant information from ElasticSearch in batches of MAX_RESULT_SIZE
-     */
-    private List<ParticipantWrapperDto> fetchParticipantEsData(Filterable filter, QueryParamsMap queryParamsMap) {
-        List<ParticipantWrapperDto> allResults = new ArrayList<ParticipantWrapperDto>();
-        int currentFrom = DEFAULT_FROM;
-        int currentTo = MAX_RESULT_SIZE;
-        while (true) {
-            // For each batch of results, add the DTOs to the allResults list
-            filter.setFrom(currentFrom);
-            filter.setTo(currentTo);
-            ParticipantWrapperResult filteredSubset = (ParticipantWrapperResult) filter.filter(queryParamsMap, new UnparsedDeserializer());
-            allResults.addAll(filteredSubset.getParticipants());
-            // if the total count is less than the range we are currently on, stop fetching
-            if (filteredSubset.getTotalCount() < currentTo) {
-                break;
-            }
-            currentFrom = currentTo;
-            currentTo += MAX_RESULT_SIZE;
-        }
-
-        return allResults;
-    }
-
```

Comment:
Moved this to the `DownloadParticipantListService` so that it is easier to test and cleaner

Reply:
great idea